BookingsApi/BookingsApi/Validations/EditJusticeUserRequestValidation.cs

```diff
@@ -0,0 +1,18 @@
+using BookingsApi.Contract.Requests;
+using FluentValidation;
+
+namespace BookingsApi.Validations;
+
+public class EditJusticeUserRequestValidation : AbstractValidator<EditJusticeUserRequest>
+{
+    public static readonly string NoUsernameErrorMessage = "Username is required";
+    public static readonly string NoIdErrorMessage = "Id is required";
+    public static readonly string NoRoleErrorMessage = "Role is required";
```

Comment:
if these strings are all static/readonly, any reason they can't just be public const?

Reply:
done